CMakeLists.txt

```diff
@@ -185,6 +185,11 @@ if (ENABLE_GOOGLE_PROFILER)
   set(LIBS ${LIBS} profiler)
 endif()
 
+# This option enables profiling individual work order performances. If enabled, 
+# at the end of each query execution, we print on stdout in CSV format the 
+# statistics about every work order that was executed. 
+option(ENABLE_WORKORDER_PROFILING "Enable profiling individual WorkOrder execution" OFF)
```

Comment:
Hi @zuyu 

Jianqiao and I had a chat about this and we agreed that gflags should be a better option instead of compile time flags. I will make that change.